src/ODC-By-1.0.xml

```diff
@@ -0,0 +1,504 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="ODC-By-1.0"
+            name="Open Data Commons Attribution License v1.0">
+      <crossRefs>
+         <crossRef>https://opendatacommons.org/licenses/by/1.0/</crossRef>
+      </crossRefs>
+    <text>
+      <titleText>
+	     <optional>##</optional>
+         <p>ODC Attribution License (ODC-By)</p>
+      </titleText>
+      <optional>###</optional>
+      <p>Preamble</p>
```

Comment:
Is the preamble optional?

Reply:
we generally don't mark the preamble as optional unless there is a good reason or definitive indication from the license steward/author to do so.